swift/Sources/SignalClient/Aes256GcmSiv.swift

```diff
@@ -3,7 +3,7 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
-import SignalFfi
+// import SignalFfi
```

Comment:
Remove (here and below)?

Reply:
Yeah, I guess. I felt weird not having it be marked at all but it's not serving a purpose in either build system at the moment.